src/Module.Server/Common/Commander/CrpgCommanderPollComponent.cs

```diff
@@ -226,6 +232,15 @@ private void OpenCommanderPollOnServer(NetworkCommunicator pollCreatorPeer, Netw
             return;
         }
 
+        CrpgUser? crpgUser = targetPeer.GetComponent<MissionPeer>().GetComponent<CrpgPeer>().User;
+        if (crpgUser?.Restrictions.Any(r => r.Type == CrpgRestrictionType.Chat) ?? false)
```

Comment:
We should probably also check whether `targetPeer.IsMuted`. If they are muted during the same game, it is unlikely that the server has fetched the updated crpgUser with the chat restriction.

Reply:
Added the local mute check too.